.github/workflows/pytest-gpu.yml

```diff
@@ -66,31 +66,21 @@ jobs:
           pgaccelinfo
         fi
         pytest --cov --cov-config=.coveragerc --cov-report=xml ${{ matrix.test_file }}
-        pytest --cov --cov-config=.coveragerc --cov-report=xml examples/seismic/elastic/elastic_example.py
-        pytest --cov --cov-config=.coveragerc --cov-report=xml examples/seismic/acoustic/acoustic_example.py
-        pytest --cov --cov-config=.coveragerc --cov-report=xml examples/seismic/viscoacoustic/viscoacoustic_example.py
-        pytest --cov --cov-config=.coveragerc --cov-report=xml examples/seismic/viscoelastic/viscoelastic_example.py
 
-        if [ "${{ matrix.name }}" == 'pytest-gpu-omp' ]; then
-            pytest --cov --cov-config=.coveragerc --cov-report=xml examples/seismic/viscoelastic/viscoelastic_example.py
-        fi
-
-    - name: Run examples
-      if: matrix.name == 'pytest-gpu-acc'
+    - name: Test examples
       run: |
-        python3 examples/seismic/elastic/elastic_example.py
-        python3 examples/seismic/elastic/elastic_example.py
-        python3 examples/seismic/tti/tti_example.py
-        python3 examples/seismic/viscoacoustic/viscoacoustic_example.py
-        python3 examples/seismic/viscoelastic/viscoelastic_example.py
+        pytest examples/seismic/elastic/elastic_example.py
+        pytest examples/seismic/elastic/elastic_example.py
```

Comment:
The elastic example is tested twice without MPI.
The acoustic example is not tested with and without MPI.